umbraco/Umbraco-CMS
# Feature in v7 based on #2682 that wasn't merged upwards to v8

Gets the feature in v7 based on #2682 that wasn't merged upwards to v8 and found some issues along the way like not filtering the mini list view properly along with not returning the content type alias in the entity controller

## Testing

* Have the starter kit installed
* Go to the products page and notice the featured products picker
* Change the featured products picker data type to not have a start node and to have a content type filter of type `product`
* Now go use the featured products picker: 
   * When you open the picker and the tree loads it will auto sync the tree to the Products level instead of just showing the home node
   * you should only be able to select products even though you can see the whole tree, also test that you cannot select People under the people node

There was fixes to this process here follow the instructions here https://github.com/umbraco/Umbraco-CMS/pull/5199 which i've merged in v7 and have manually merged these changes up to v8.
